packages/react-components/src/types/ReactionTypes.ts

```diff
@@ -48,3 +48,10 @@ export interface ReactionResources {
    */
   surprisedReaction?: ReactionSprite;
 }
+
+//* @conditional-compile-remove(reaction) */
+/**
+ * Options for overlay mode for reaction rendering
+ * @internal
+ */
+export type OverlayModeTypes = 'grid-tiles' | 'screen-share' | 'content-share' | 'together-mode';
```

Comment:
lets leave together mode out until that is a feature in development. we don't want it to be part of the API if its something we don't support just yet

Reply:
Keeping the type here since it will come in handy for other features i.e. together mode in future.